File: Site/src/app/model/evento.ts
import { Sala } from './sala';
import { Usuario } from './Usuario';

export class Evento {

    public CodEvento: number;
    public Nome: string;
    public CodUsuario: Usuario;
    public CodSala: Sala;
    public DataInicio: string;
    public DataFim: string;

    // constructor() {
    //     this.CodEvento = null;
    //     this.CodEvento = '';
    // }
}

File: Api/GerenciaSalaReuniao.Dominio/Entidade/Evento.cs

using System;

namespace GerenciaSalaReuniao.Dominio.Entidade
{
    public class Evento
    {
        public int CodEvento { get; set; }
        public string Nome { get; set; }
        public int CodSala { get; set; }
        public int CodUsuario { get; set; }
        public DateTime DataInicio { get; set; }
        public DateTime DataFim { get; set; }

        public virtual Sala Sala { get; set; }
        public virtual Usuario Usuario { get; set; }
    }
}
